<ui version="4.0" >
 <class>KopeteMetaLVIPropsWidget</class>
 <widget class="QWidget" name="KopeteMetaLVIPropsWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>441</height>
   </rect>
  </property>
  <property name="toolTip" >
   <string/>
  </property>
  <layout class="QVBoxLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>&amp;General</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QGroupBox" name="grpAddressbook" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title" >
          <string>Address Book Link</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="leftMargin" >
           <number>11</number>
          </property>
          <property name="topMargin" >
           <number>11</number>
          </property>
          <property name="rightMargin" >
           <number>11</number>
          </property>
          <property name="bottomMargin" >
           <number>11</number>
          </property>
          <item>
           <widget class="Kopete::UI::AddressBookLinkWidget" native="1" name="widAddresseeLink" />
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="spacing" >
             <number>6</number>
            </property>
            <property name="leftMargin" >
             <number>0</number>
            </property>
            <property name="topMargin" >
             <number>0</number>
            </property>
            <property name="rightMargin" >
             <number>0</number>
            </property>
            <property name="bottomMargin" >
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="btnExportKABC" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="toolTip" >
               <string>Export contact's details to the KDE Address Book</string>
              </property>
              <property name="text" >
               <string>E&amp;xport Details...</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>107</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnImportKABC" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="toolTip" >
               <string>Import contacts from the KDE Address Book</string>
              </property>
              <property name="text" >
               <string>&amp;Import Contacts</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="buttonGroup1" >
         <property name="title" >
          <string>Display Name Source</string>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" colspan="4" >
           <widget class="QRadioButton" name="radioNameKABC" >
            <property name="text" >
             <string>Use address book &amp;name (needs address book link)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2" >
           <widget class="QRadioButton" name="radioNameContact" >
            <property name="text" >
             <string>From contact:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>16</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="3" >
           <widget class="QComboBox" name="cmbAccountName" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip" >
             <string>Contact with which to synchronize the displayname.</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QRadioButton" name="radioNameCustom" >
            <property name="text" >
             <string>Cus&amp;tom:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>51</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="3" >
           <widget class="QLineEdit" name="edtDisplayName" />
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="buttonGroup2" >
         <property name="title" >
          <string>Photo Source</string>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QRadioButton" name="radioPhotoKABC" >
            <property name="text" >
             <string>U&amp;se address book photo (needs address book link)</string>
            </property>
           </widget>
          </item>
          <item rowspan="3" row="0" column="1" >
           <widget class="QLabel" name="photoLabel" >
            <property name="minimumSize" >
             <size>
              <width>96</width>
              <height>96</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>96</width>
              <height>96</height>
             </size>
            </property>
            <property name="baseSize" >
             <size>
              <width>96</width>
              <height>96</height>
             </size>
            </property>
            <property name="frameShape" >
             <enum>QFrame::Box</enum>
            </property>
            <property name="lineWidth" >
             <number>1</number>
            </property>
            <property name="text" >
             <string>Photo</string>
            </property>
            <property name="scaledContents" >
             <bool>false</bool>
            </property>
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QRadioButton" name="radioPhotoContact" >
              <property name="text" >
               <string>From contact:</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>21</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QComboBox" name="cmbAccountPhoto" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip" >
               <string>Contact with which to synchronize the displayname.</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0" >
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QRadioButton" name="radioPhotoCustom" >
              <property name="text" >
               <string>Custom:</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>181</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnChoosePhoto" >
              <property name="text" >
               <string>Choose...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnClearPhoto" >
              <property name="maximumSize" >
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="text" >
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkSyncPhoto" >
         <property name="text" >
          <string>S&amp;ync photo to address book</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="TabPage" >
      <attribute name="title" >
       <string>Ad&amp;vanced</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QGroupBox" name="grpIcons" >
         <property name="title" >
          <string>Icons</string>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" colspan="5" >
           <widget class="QCheckBox" name="chkUseCustomIcons" >
            <property name="toolTip" >
             <string>Check to set custom icons for this contact</string>
            </property>
            <property name="text" >
             <string>Use custom status &amp;icons</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="lblOnline" >
            <property name="text" >
             <string>&amp;Online:</string>
            </property>
            <property name="buddy" >
             <cstring>icnbOnline</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="KIconButton" name="icnbOnline" >
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="2" >
           <widget class="QLabel" name="lblOffline" >
            <property name="text" >
             <string>O&amp;ffline:</string>
            </property>
            <property name="buddy" >
             <cstring>icnbOffline</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="3" >
           <widget class="KIconButton" name="icnbOffline" >
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="4" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="lblAway" >
            <property name="text" >
             <string>Awa&amp;y:</string>
            </property>
            <property name="buddy" >
             <cstring>icnbAway</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="KIconButton" name="icnbAway" >
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="2" >
           <widget class="QLabel" name="lblUnknown" >
            <property name="text" >
             <string>Un&amp;known:</string>
            </property>
            <property name="buddy" >
             <cstring>icnbUnknown</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="3" >
           <widget class="KIconButton" name="icnbUnknown" >
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>91</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Kopete::UI::AddressBookLinkWidget</class>
   <extends>QWidget</extends>
   <header>addressbooklinkwidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>btnExportKABC</tabstop>
  <tabstop>btnImportKABC</tabstop>
  <tabstop>radioNameKABC</tabstop>
  <tabstop>radioNameContact</tabstop>
  <tabstop>cmbAccountName</tabstop>
  <tabstop>radioNameCustom</tabstop>
  <tabstop>edtDisplayName</tabstop>
  <tabstop>radioPhotoKABC</tabstop>
  <tabstop>radioPhotoContact</tabstop>
  <tabstop>cmbAccountPhoto</tabstop>
  <tabstop>radioPhotoCustom</tabstop>
  <tabstop>btnChoosePhoto</tabstop>
  <tabstop>btnClearPhoto</tabstop>
  <tabstop>chkSyncPhoto</tabstop>
  <tabstop>chkUseCustomIcons</tabstop>
  <tabstop>icnbOnline</tabstop>
  <tabstop>icnbOffline</tabstop>
  <tabstop>icnbAway</tabstop>
  <tabstop>icnbUnknown</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
